airbyte-integrations/connectors/source-mailchimp/source_mailchimp/schemas/list_members.json

```diff
@@ -0,0 +1,407 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": ["object", "null"],
+    "title": "Members List",
+    "description": "Information about members in a specific list.",
+    "properties": {
+      "id": {
+        "type": "string",
```

Comment:
For each of the types declared here, can you also add null to them so that the connector can continue to work even if there are api changes/regressions?